src/Workspaces/Core/Portable/FindSymbols/FindReferences/FindReferenceCache.cs

```diff
@@ -19,36 +19,27 @@
 
 namespace Microsoft.CodeAnalysis.FindSymbols
 {
-    // TODO : this can be all moved down to compiler side.
     internal static class FindReferenceCache
     {
-        private static readonly ReaderWriterLockSlim s_gate = new();
```

Comment:
this file is pretty heavily modified.  looking entirely at teh 'after' side might be the most helpful.